packages/p2p/src/stores/order-store.js

```diff
@@ -198,6 +252,17 @@ export default class OrderStore {
         this.hideDetails(false);
     }
 
+    @action.bound
+    setOrderDetails(response) {
+        if (!response.error) {
```

Comment:
```suggestion
        if (response && !response.error) {
```

Reply:
it should be fixed already, look at the commit 'dont kill me' hahah

Reply:
oh wait this wasn't changed

Reply:
Adding a separate `if (response)` condition on top would be better since the `if (response && !response.error) {` condition will cause the else condition to run when there's no response.